### PR TITLE
Paragraph title update: Update replay-debugger.md

### PR DESCRIPTION
**Pull request by eyewell**
### What does this PR do?
Changed line 54, since the enabling of automatic logging is not based on whether the org is the "default dev org", but instead, on whether the org is a DE or Scratch Org.

### What issues does this PR fix or reference?
#<Insert GitHub Issue>, @<Insert GUS WI>@

### Functionality Before
<insert gif and/or summary>

### Functionality After
<insert gif and/or summary>

### CODE REVIEW

File: docs/_articles/en/apex/replay-debugger.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ Before you generate a debug log for replay debugging, set breakpoints and checkp
         - Or, to convert an existing breakpoint into a checkpoint, right-click the breakpoint, and select **Edit Breakpoint** \| **Expression**. Set the expression to `Checkpoint`.
     1.  To upload your checkpoints to your org to collect heap dump information, open the command palette, and run **SFDX: Update Checkpoints in Org**.
 
-### Set Up an Apex Replay Debugger Session for a Scratch Org or a Default Development Org
+### Set Up an Apex Replay Debugger Session for a Scratch Org or a Developer Edition Org
```
Comment (brpowell) [inline docs/_articles/en/apex/replay-debugger.md:54]: FYI @sbudhirajadoc small doc update on the doc site

**Comment by brpowell** [comment]
Thanks for the update @eyewell, as soon as we get this approved and you confirm your identity with the [CLA bot](https://github.com/forcedotcom/salesforcedx-vscode/pull/3182#issuecomment-828719131) we can get this merged.

File: docs/_articles/en/apex/replay-debugger.md (hunk 1 above)
Comment (sbudhirajadoc) [reply docs/_articles/en/apex/replay-debugger.md:54]: the change LTGM.

**Review by sbudhirajadoc — APPROVED** [review]

**Comment by randi274** [comment]
@eyewell it looks like this dropped off our radar for a bit, but when you get a chance can you complete the [CLA](https://docs.google.com/forms/d/e/1FAIpQLSfB_vIrwFLweXIGt7ID98JhISCU2zm2ErOugAnGAcCvAzr7NQ/viewform)?

**Comment by eyewell** [comment]
Done! sorry for the delay!

**Comment by gbockus-sf** [comment]
Looks like this is no longer valid after the replay debugger test update.

**Pull request closed without merging**